modules/govuk/manifests/apps/govuk_delivery.pp

```diff
@@ -78,15 +78,13 @@
   }
 
   govuk::app::envvar::mongodb_uri { $app_name:
-    ensure   => 'absent',
```

Comment:
I think we can leave this line, this is only to configure the env.d file

Reply:
CI fails with this in here (and the redis one below) e.g. https://ci.integration.publishing.service.gov.uk/job/govuk-puppet/job/more-govuk-delivery-removal/7/console